doc/interfaces/tracking_flow.rst

```diff
@@ -0,0 +1,180 @@
+getter which draws the direction from the distribution. Therefore, the maximum
+deterministic direction getter is equivalent to the probabilistic direction
+getter returning always the maximum value of the distribution.
+
+Deterministic maximum fiber tracking is an alternative to EuDX deterministic
+tractography and unlike EuDX does not follow the peaks of the local models but
+uses the entire orientation distributions.
+
+We will first create the directory in which to save the output tractogram file
+(e.g.: ``det_tracking_output``)::
+
+    mkdir det_tracking_output
+
+Then, to perform the deterministic tracking we will run the ``dipy_track``
+command as::
+
+    dipy_track recons-csa/peaks.pam5 stop_file/mask.nii.gz wm_mask/brain_mask.nii.gz --seed_density 2 --tracking_method "det" --out_dir "det_tracking_output"
+
+Probabilistic Tracking
+**********************
+
+Probabilistic fiber tracking is a way of reconstructing white matter connections
+using diffusion MR imaging. Like deterministic fiber tracking, the probabilistic
+approach follows the trajectory of a possible pathway step by step starting at a
+seed, however, unlike deterministic tracking, the tracking direction at each
+point along the path is chosen at random from a distribution. The distribution
+at each point is different and depends on the observed diffusion data at that
+point. The distribution of tracking directions at each point can be represented
+as a probability mass function (PMF) if the possible tracking directions are
+restricted to discrete numbers of well distributed points on a sphere.
+
+We will first create the directory in which to save the output tractogram
+file (e.g.: ``det_tracking_output``)::
+
+    mkdir det_tracking_output
+
+Then, to perform the deterministic tracking we will run the ``dipy_track``
+command as::
+
+    dipy_track recons-csa/peaks.pam5 stop_file/mask.nii.gz wm_mask/brain_mask.nii.gz --seed_density 2 --tracking_method "det" --out_dir "det_tracking_output"
+
+ClosestPeaks Tracking
+*********************
+
+We will first create the directory in which to save the output tractogram
+file (e.g.: ``det_tracking_output``)::
+
+    mkdir det_tracking_output
+
+Then, to perform the deterministic tracking we will run the ``dipy_track``
+command as::
+
+    dipy_track recons-csa/peaks.pam5 stop_file/mask.nii.gz wm_mask/brain_mask.nii.gz --seed_density 2 --tracking_method "cp" --out_dir "cp_tracking_output"
+
+------------------------------------
+Particle Filtering Tracking PAM Flow
+------------------------------------
+
+Particle Filtering Tracking (PFT) uses tissue partial volume estimation (PVE)
+to reconstruct trajectories connecting the gray matter, and not incorrectly
+stopping in the white matter or in the corticospinal fluid. It relies on a
+stopping criterion that identifies the tissue where the streamline stopped.
+If the streamline correctly stopped in the gray matter, the trajectory is kept.
```

Comment:
If the streamline correctly stopped in the gray matter, (...): If the streamline reaches the gray matter, (...)

Reply:
Addressed in 029f8a5